meilisearch-core/benches/search_benchmark.rs

```diff
@@ -4,11 +4,13 @@ extern crate assert_matches;
 
 use std::sync::mpsc;
 use std::path::Path;
-use std::fs;
+use std::{fs, fs::File, io::BufReader};
```

Comment:
Split this into 2 lines. 
Use only `File` instead of `fs::File` at the line 51.
```suggestion
use std::fs::File;
use std::io::BufReader;
```